src/data/resolvers/mutations/customers.js

```diff
@@ -48,6 +48,34 @@ const customerMutations = {
   async customersEditCompanies(root, { _id, companyIds }) {
     return Customers.updateCompanies(_id, companyIds);
   },
+
+  /**
+   * Merge customers
+   * @param {string[]} customerIds - First customer to merge
+   * @param {Object} newCustomer - Newly created customer infos
+   * @return {Promise} Customer object
+   */
+  async customersMerge(root, { customerIds, newCustomer }) {
```

Comment:
rename newCustomer variable name. It is confusing. This is not actual customer object